app/src/main/java/com/android/voyageur/model/trip/Trip.kt

```diff
@@ -0,0 +1,33 @@
+package com.android.voyageur.model.trip
+
+import com.android.voyageur.model.location.Location
+import com.google.firebase.Timestamp
+
+data class Trip(
+    val id: String,
+    val creator: String,
+    val participants: Array<String>,
+    val description: String,
+    val name: String,
+    val location: Location,
```

Comment:
What if someone is doing a trip to multiple locations? For example multiple cities within the same country? Or multiple countries?
Let's maybe make this `val locations: Array<Location>` to support these cases, wdyt?

Reply:
Yes you are right, I made it an array.